### ISSUE TITLE
Validation Buttons Missing

### ISSUE THREAD

**Issue by ThatOneGoat** [ProjectSidewalk/SidewalkWebpage#2092]
I was testing on mobile when I ran into this issue on both my branch and the prod server:
![MissingValidateButtons](https://user-images.githubusercontent.com/37346427/80895190-0d575900-8c97-11ea-814e-f54a028758c0.png)

The strange thing is I can't reproduce this issue. I suspect it was because of some loading race conditions with how elements are rendered on the mobile page, but some investigating needs to be done first before concluding that.

**Comment by ThatOneGoat** [comment]
Another note, I'm using an IPhone 6

**Comment by misaugstad** [comment]
duplicate of #1881
